src/bttrx_fsm.cpp

```diff
@@ -242,6 +250,9 @@ void BTTRX_FSM::handleIncomingMessage()
 	wt32i_.getIncomingMessage(&msg);
 
 	switch (msg.msg_type) {
+	case kSETTING_CALLSIGN:
+		bttrx_control_.storeSetting(kCallsign, splitString(msg.msg)[3]);
+		break;
```

Comment:
Please remove. This switch handles incoming messages from the wt32i module. There is no CALLSIGN setting in the iWrap specification.

Reply:
Solved with https://github.com/phl0/firmware/commit/9bece9392f208d0897e485a552e37f3b87ffc408. (Will rebase to 1.1.0 dev branch as soon as all issues are done).